docs/spec/normative_examples.md

```diff
@@ -493,79 +496,107 @@ metadata:
   name: my-service
   generation: 1
   ...
-spec: 
+spec:
   ...  # same as before
 status:
   # the latest created and ready to serve.
   latestReadyRevisionName: abc
-  # latest created revision 
+  # latest created revision
   latestCreatedRevisionName: abc
   observedGeneration: 1
 ```
 
 
-## 3) Manual rollout of a new Revision - config change only
+## 3) Managed release of a new Revision - config change only
 
-**_Scenario_**: User updates configuration with new runtime arguments
-  (env var change) to an existing service, tests the revision, then
-  proceeds with a manually controlled rollout to 100%
+**_Scenario_**: User updates configuration with new runtime arguments (env var
+  change) to an existing service, tests the revision, then proceeds with a
+  human-controlled rollout to 100%.
 
 ```
-$ knative rollout --service my-service strategy manual
+$ knative release --service my-service strategy release
+
+$ knative revisions list --service my-service
+Route           Traffic  Id    Date                Deployer      Git SHA
+current,latest  100%     def   2018-01-18 20:34    user1         a6f92d1
+                         abc   2018-01-17 10:32    user1         33643fc
 
 $ knative deploy --service my-service --env HELLO="blurg"
 [...]
+Deployed revision ghi to https://latest.my-service.default.mydomain.com
+You can begin rolling out this revision with [knative release begin ghi]
 
 $ knative revisions list --service my-service
-Name     Traffic  Id   Date                Deployer     Git SHA
-next     0%       v3   2018-01-19 12:16    user1        a6f92d1
-current  100%     v2   2018-01-18 20:34    user1        a6f92d1
-                  v1   2018-01-17 10:32    user1        33643fc
+Route           Traffic  Id    Date                Deployer     Git SHA
+latest          0%       ghi   2018-01-19 12:16    user1        a6f92d1
+current         100%     def   2018-01-18 20:34    user1        a6f92d1
+                         abc   2018-01-17 10:32    user1        33643fc
+
+$ knative release --service my-service begin ghi
 
-$ knative rollout next percent 5
+$ knative revisions list --service my-service
+Route           Traffic  Id    Date                Deployer     Git SHA
+next,latest     0%       ghi   2018-01-19 12:16    user1        a6f92d1
```

Comment:
'next' was changed to 'candidate' so might be worth a s/next/candidate/g

Otherwise this lgtm to close out.